Program/Params.cpp

```diff
@@ -13,13 +13,14 @@ Params::Params(
 	double durationLimit,
 	int nbVeh,
 	bool isDurationConstraint,
+	bool useCoordinates,
```

Comment:
I liked the idea of a useSwapStar parameter. By default, we set it to false when we have no coordinates and true otherwise, but some other circumstances could lead the user to define it as false as part of a calibration process (e.g. extremely large problems).

Reply:
I see your point. What about then we have an option parameter like

`useSwapStar = 0, 1, 2` 

with 

`0: Automatic`
`1: UseSwapStar`
`2: DoNotUseSwapStar`

or

`0: DoNotUseSwapStar`
`1: UseSwapStar`
`2: Automatic`

Reply:
Hum... I'm not certain three values are needed.
I would simply set it to
useSwapStar = 0, 1 (default to 1)
Moreover, adding a message in case no coordinates are available and useSwapStar = 1, saying that Swap Star has been deactivated since no coordinates have been provided.

Reply:
I was thinking of a scenario like:


When dist_mtx is provided, but not x, y coordinates. The user didn't specify `useSwapStar` option. The default value of `useSwapStar` is `on`.  But, in this case, the default behavior should be NOT to use the SWAP* search; therefore should be changed to `off` inside the program. 

I thought the above case is quite strange. That's why I came up with `Automatic`.  But if we just specify SWAP* is usable only when the coordinates are provided, it should be no problem. Also, checking the usefulness of the coordinates is the user's responsibility.

Reply:
One more checking: do you think negative values can be used for coordinates? Any potential issues?

Reply:
I thought the above case is quite strange. That's why I came up with Automatic. But if we just specify SWAP* is usable only when the coordinates are provided, it should be no problem. Also, checking the usefulness of the coordinates is the user's responsibility.
>> I think this is a good approach

Reply:
No problem having negative coordinates, to my knowledge, though I believe this has been rarely tested.

Reply:
Thanks. I'll make changes accordingly.